app.py

```diff
@@ -25,8 +28,15 @@ def create_movies_form():
 
 @app.post('/movies')
 def create_movie():
-    # TODO: Feature 2
+    # TODO: Feature 2 - COMPLETE
     # After creating the movie in the database, we redirect to the list all movies page
+    movie_repository.create_movie(request.form.get('movie'), request.form.get('director'), request.form.get('rating'))
+
+#if the movie repository isn't working, uncomment the 3 lines below and use that instead.
```

Comment:
these commented things are no longer needed

Reply:
Got it.